packages/webpack-cli/bin/cli.js

```diff
@@ -20,6 +20,11 @@ if (packageExists('webpack')) {
 } else {
     promptInstallation('webpack', () => {
         error(`It looks like ${yellow('webpack')} is not installed.`);
-    });
+    })
+        .then(() => info(`${yellow('webpack')} was installed sucessfully.`))
```

Comment:
Can we have tests?

Reply:
CI already installs webpack before running tests.